### PR TITLE
Quality of life fixes.

### PR DESCRIPTION
**Pull request by lramos15**
Fixes a bunch of maintenance things that needed to be done for this project.

1. Updates to using Node 16 as Node 14 is EOL
2. Updates the DB schema to UTF-8 so we can support and render emojis properly. (@ddbruce Please update the production database ❤️ )
3. Bumps a bunch of old packages
4. Removes a few ES lint rules that I found nothing but annoying and slowed down the dev experience. If you feel differently please let me know.

Some open questions:
1. I couldn't reproduce the flashing locally, hoping that bumping the socket.io library fixes that if not I'll leave it running on my machine for longer to figure out.
2. Why do we serve moment from a CDN but not the other modules if we have them installed in the node_modules folder anyways of the project.
3. There are no dev steps. I was able to piece together a working environment but I have no clue what the chore token does, the website DB had to be grabbed from the website repo, and the experience is just well terrible.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -28,7 +29,7 @@ app.use(bodyParser.json());
 
 app.use('/public', express.static(path.join(__dirname, 'public')));
 
-app.get('/suncalc.js', (req, res) => {
+app.get('/suncalc.js', (_, res) => {
```
Comment (ddbruce) [inline server.js:32]: What's going on here and the other, similar `app.get`s?

**Comment by ddbruce** [comment]
@lramos15 prod database is already set to `utf8mb4_unicode_ci`; that should be acceptable, and afaik we've had emojis working previously. Maybe I'm misremembering.

Please run a `npm version patch` or `minor` depending on what level you think this is. The pr-version-check failed.

File: server.js (hunk 1 above)
Comment (lramos15) [reply server.js:32]: This is just standard notation for an unused variable in a callback. It gets better highlighting in the editor

Comment (ddbruce) [reply server.js:32]: Ah ok, that makes sense now

**Review by ddbruce — CHANGES_REQUESTED** [review]
Looks like you need to bump `package.json`, not just the lock file.

**Comment by lramos15** [comment]
> ~Looks like you need to bump `package.json`, not just the lock file.~ never mind

Impatient. I didn't even click to re-request your review yet :p

**Review by ddbruce — APPROVED** [review]

**Pull request merged**